packages/react/src/hooks/useMembers.ts

```diff
@@ -100,10 +100,17 @@ export default function useMembers(roomSession: Video.RoomSession | null): {
     }
     function onRoomJoined(e: any) {
       selfId.current = e.member_id;
-      console.log(selfId.current);
       setMembers(addMethods(e.room_session.members));
     }
-    roomSession.on("room.joined", onRoomJoined);
+    if (roomSession.active) {
+      (async () => {
+        selfId.current = roomSession?.memberId ?? null;
+        const members = (await roomSession?.getMembers())?.members;
+        if (members) setMembers(addMethods(members));
+      })();
+    } else {
+      roomSession?.on("room.joined", onRoomJoined);
```

Comment:
I would subscribe to `room.joined` anyway; with Broadcasting we may start getting `room.joined` events even after joining for the first time